testproject/requirements.txt

```diff
@@ -1,2 +1,4 @@
 pytz==2020.1
 django-csp==3.6
+boto3=1.14.20
```

Comment:
I got an error trying to install the `requirements.txt`. I think `==` is expected.

Reply:
Will fix soon.